src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenu.cshtml

```diff
@@ -1,7 +1,7 @@
 @inject IAuthorizationService AuthorizationService
 <div class="dropdown">
     <form asp-route-area="OrchardCore.Users" asp-controller="Account" asp-action="LogOff" method="post" class="form-inline" class="no-multisubmit">
-        <a class="ml-3 dropdown-toggle" id="navbarDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" role="button" data-toggle-tooltip="tooltip" title="@User.Identity.Name">
+        <a class="ml-3 dropdown-toggle" id="navbarDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" data-toggle-tooltip="tooltip" title="@User.Identity.Name">
```

Comment:
Why not use a `<button>` instead? Also in src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml.

Also see: https://stackoverflow.com/questions/57398579/when-and-when-not-to-use-role-button

Reply:
I was thinking about using `<button>` here, but I need to change a lot of CSS for that, and I went with this because the change would be minimal.

I will replace it with buttons. If I understand it correctly, [this](https://github.com/OrchardCMS/OrchardCore/blob/67256b454fe9cdb3f14ffc91bd3ed8ce9bcfbbd1/src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml#L57) can stay like this, right? It is a link, after all.

Reply:
Yes, that looks good. The principle is that if it has button behavior, which this element does, then it's best if it's actually a `button`.